nengo_gui/static/side_menu.css

```diff
@@ -0,0 +1,128 @@
+
+.sidenav-container {
+    height: calc(100% - 120px);
+    width: 250px;
+    position: fixed;
+    z-index: 9999;
+    top: 34px;
+    left: 0;
+    background-color: #eee;
+    overflow-x: hidden;
+    overflow-y: hidden;
+    transition: 0.5s;
+    border-right: 1px solid #4D4D4D;
+    font-size: 0px;
+    color: #333;
+    font-family: arial;
+    transform: translate(-250px);
+}
+
+.sidenav-container .tab-content{
+  font-size: 15px;
```

Comment:
Inconsistent tab-width here... we don't really enforce any CSS style though so probably not worth fixing.